requirements.txt

```diff
@@ -7,3 +7,4 @@ Pillow
 requests
 bokeh
 protobuf
+psutil
```

Comment:
shouldn't this be in `requirements_dev.txt` as this is only used for developers doing unit tests?  If we build BSK wheels we don't need `plutil` to be included in the BSK wheel requirements.